app/assets/stylesheets/application.css

```diff
@@ -13,7 +13,7 @@
  *= require_tree .
  *= require_self
  */
-
+ @import "font-awesome";
```

Comment:
Do we use any font-awesome-rails icons here? Since the icons here are in assets folder..

Reply:
have removed the import statement